src/components/TodoItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
+    <div className="todo">
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked
+        />
+      </label>
+
+      <span
+        className="todo__title"
+      >
+        {todo.title}
```

Comment:
You can pass only the title to this component. Seems that you don't need a whole todo here